realm/realm-annotations-processor/src/test/resources/io/realm/AllTypesRealmProxy.java

```diff
@@ -635,7 +635,7 @@ public static String getTableName() {
             } else if (name.equals("columnMutableRealmInteger")) {
                 Long val = null;
                 if (reader.peek() != JsonToken.NULL) {
-                    val = reader.nextLong()
+                    val = reader.nextLong();
```

Comment:
How did this compile before? 🤔

Reply:
No idea.
I guess Google's `compile-testing` library does not see the semi-colon.